4.classify-features/classification-utils.py

```diff
@@ -35,23 +36,43 @@ def classify_plate_cells(classifier, plate_load_path: pathlib.Path) -> pd.DataFr
     # combine both dictionaries
     plate_dtypes = {**metadata_dtypes, **feature_dtypes}
 
-    prediction_classes = classifier.classes_
-
-    print(f"Loading feature data for {plate_load_path.name}...")
-    # specify datatypes for all columns to make loading more efficient
-
     # load in csv with the specified column names/datatypes
     plate_data = pd.read_csv(
         plate_load_path, compression="gzip", dtype=plate_dtypes, low_memory=True
     )
 
-    print(f"Classifying feature data for {plate_load_path.name}...")
-
     # combine metadatas and classifications dataframes
     cell_metadatas = plate_data[metadata_cols].reset_index(drop=True)
     cell_features = plate_data[feature_cols].values
+    prediction_classes = classifier.classes_
     cell_classifications = pd.DataFrame(
         classifier.predict_proba(cell_features),
         columns=prediction_classes,
```

Comment:
Minor suggestion: consider referencing `classifier.classes_` directly to clarify where the columns come from and conserve variable usage within the function.

```suggestion
    cell_classifications = pd.DataFrame(
        classifier.predict_proba(cell_features),
        columns=classifier.classes_,
```